s3.go

```diff
@@ -40,32 +42,30 @@ func PutObject(key, bucket, s3Class string) error {
 
 // GetObject - Get object from s3 bucket
 func GetObject(key, bucket string) error {
-	cfg, err := config.LoadDefaultConfig(context.TODO())
-	session := s3.NewFromConfig(cfg)
-
 	i := &s3.GetObjectInput{
 		Bucket: aws.String(bucket),
 		Key:    aws.String(key),
 	}
 
-	size, err := session.GetObject(context.TODO(), i)
-
-	log.Printf("Cache downloaded successfully, containing %d bytes", size)
```

Comment:
My tests initially failed because it turned out that `session.GetObject` doesn't return a `size` int, it actually returns a full-fledged object. There's no direct way to get the size of the object from what's returned, so I tried to do the closest thing possible - displaying the Content Range if it exists.